cmd/agent/checkers.go

```diff
@@ -28,6 +30,10 @@ import (
 type config struct {
 	// role is the current agent's role
 	role agent.Role
+	// serfRPCAddr is the Serf RPC endpoint address
+	serfRPCAddr string
+	// serfRPCName is the Serf RPC endpoint address
```

Comment:
Looks like the comment was copied from the above field.

Reply:
fixed.